azext_iot/central/_help.py

```diff
@@ -1302,3 +1303,118 @@ def _load_central_monitors_help():
         type: command
         short-summary: Get the device twin from IoT Hub.
     """
+
+
+def _load_central_edge_help():
+    helps[
+        "iot central device edge"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central edge devices
+    """
+
+    helps[
+        "iot central device edge list"
+    ] = """
+        type: command
+        short-summary: Get the list of edge devices for an IoT Central application.
+        examples:
+        - name: List all edge devices in an application, sorted by device Id (default)
+          text: >
+            az iot central device edge list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central device edge show"
+    ] = """
+        type: command
+        short-summary: Get an IoT Edge device for an IoT Central application.
+        examples:
+        - name: Get an IoT Edge device in an application.
+          text: >
+            az iot central device edge show
+            --app-id {appid}
+            --device-id {deviceId}
+    """
+
+    helps[
+        "iot central device edge get-children"
```

Comment:
If there is going to be more device children/leaf-node operations then recommend, `az iot central device edge children *`  i.e. `az iot central device edge children list`

As a reference or data point here is how iot hub device children commands are organized

![image](https://user-images.githubusercontent.com/28658112/155585606-c2a49958-0dc8-4cc3-bccf-7c17273d15b4.png)

And related for device relationships, here are parent device commands organization

![image](https://user-images.githubusercontent.com/28658112/155585759-398701a9-ef27-45f0-8df8-928fe9306f29.png)

Reply:
based on the suggestion, we moved to "children" subgroup.
- az iot central device edge children list
- az iot central device edge children add
- az iot central device edge children remove

We prefer not to have a "parent" section for IoTC